pkgs/by-name/fl/fluent-bit/package.nix

```diff
@@ -19,13 +22,18 @@ stdenv.mkDerivation (finalAttrs: {
   src = fetchFromGitHub {
     owner = "fluent";
     repo = "fluent-bit";
-    rev = "v${finalAttrs.version}";
+    rev = "refs/tags/v${finalAttrs.version}";
     hash = "sha256-BnrULjcWVBAOHfxlmd1RTQ8gfwlfZcwrUyLU27/9Z3M=";
   };
 
-  # optional only to avoid linux rebuild
+  # Optional only to avoid Linux rebuild.
   patches = lib.optionals stdenv.hostPlatform.isDarwin [ ./macos-11-sdk-compat.patch ];
```

Comment:
If we can't fix the darwin build, just mark it as broken for now. Not sure how many people use this on darwin anyway.

Reply:
When [staging-next](https://github.com/NixOS/nixpkgs/pull/361878) merges, the default SDK on x86_64-darwin will be the same as aarch64-darwin (11.3). Unless you’re planning to backport to 24.11, I’d just leave x86_64-darwin broken for now and not mark it broken because the default SDK bump will fix it soon.

Reply:
If Clang changed to 19.1.5, then that should also include the SDK changes. They were all part of the same cycle.